modules/ROOT/pages/release-notes/release-notes5013.adoc

```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
 
 NOTE: The previous hex code color format without the "#" is still accepted as valid values.
 
-For more information on the `color_map` setting, refer to the link:{baseurl}/configure/content-appearance/#color_map[documentation].
+For more information on the `color_map` setting, refer to the xref:configure/content-appearance.adoc#color_map[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
 The updated TinyMCE 5.0.13 editor comes with numerous changes to the previous versions. A comprehensive list of the updates
-and fixed issues is available in the link:{baseurl}/changelog/#version5013august62019[changelog].
+and fixed issues is available in the xref:changelog.adoc#version5013august62019[changelog].
```

Comment:
This xref doesn't resolve as the `version5013august62019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version5013august62019]]
== 5.0.13 - 2019-08-06
```

Reply:
done